src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -151,6 +151,13 @@
                 az ad app update --id e042ec79-34cd-498f-9d9f-123456781234 --set groupMembershipClaims=All
 
 """
+helps['ad app grant'] = """
+    type: command
+    short-summary: Grant the app OAuth2 permissions for the respective apps
+    examples:
+        - name: update a native application with OAuth2 permissions with space separated clientIds for AAD apps with TTL of 1 year
+          text: az ad app update --id e042ec79-34cd-498f-9d9f-1234234 --appid a0322f79-57df-498f-9d9f-12678 --expires 1
```

Comment:
I believe the command should be `grant`, instead of `update`

Reply:
Changed to grant and reworked for new command ```ad app permission grant```